### ISSUE TITLE
Definir arquivos mínimos para execução

### ISSUE THREAD

**Issue by AndreaSanchezTapia** [Model-R/Back-end#2]
Temos o exemplo inteiro da Mata Atlântica aqui e isso faz com que cada rodada de teste seja muito longa e com funções paralelas que talvez sejam repetidas com a base de dados (limpeza, processamento de camadas preditoras). 

Proponho:
- [ ] tirar os dados de anfíbios e aves.  - deixar `FLORA_occs_final10Km.csv`
- [ ] tirar tudo o referente a limpeza taxonômica
- [ ] tirar tudo o referente a criação de eigenvariables - mantendo as camadas.
- [ ] modificar a resolução das camadas de teste para 10 x 10 km
- [ ] tirar os scripts que não estão rodando específicamente a modelagem - deixar apenas `script.R`

**Comment by FelipeSBarros** [comment]
Concordo. Sugiro fazer uma limpeza nas funções que estão no arquivo read.eval1.R
Deve ter coisa desnecessária lá. Além de trocar o nome do arquivo para algo como "toolbox. R"

**Comment by FelipeSBarros** [comment]
O [arquivo](https://github.com/Model-R/Back-end/blob/master/ARF_sp_DataPreparation) `ARF_sp_DataPreparation` também pode ser removido pois foi criado para a análise da Mata Atlantica...

**Comment by gmgall** [comment]
Pré-requisito da issue #8.

**Comment by AndreaSanchezTapia** [comment]
podemos manter um subconjunto de dados como `data()` do pacote, para rodar exemplos (Zygia e mais alguém) e guardar esse `toolbox.R` como funções do pacote (se vale a pena)

**Comment by gmgall** [comment]
O Hadley Wickham tem uma abordagem parecida com essa do `toolbox.R` para criar pacotes. Do [capítulo sobre namespaces](http://r-pkgs.had.co.nz/namespace.html#exports):

> I believe that packages that have a wide audience should strive to do one thing and do it well. All functions in a package should be related to a single problem (or a set of closely related problems). Any functions not related to that purpose should not be exported. For example, most of my packages have a `utils.R` file that contains many small functions that are useful for me, but aren’t part of the core purpose of those packages. I never export these functions.

No pacote que criei, por exemplo, temos as funções `do_algortimo()` exportadas e funções utilitárias como `cropModel()` e `createBuffer()` não exportadas. Essas funções poderiam estar todas num `tools.R` ou `utils.R`.

**Comment by diogosbr** [comment]
Vou elaborar esse conjunto essa semana OK?
Umas duas ou três espécies tá bom, né?

**Comment by AndreaSanchezTapia** [comment]
@diogosbr, acho que não precisa ser um conjunto aparte do que já tem na pasta.
